fronts-client/README.md

```diff
@@ -179,3 +179,15 @@ We already handle all of our persistence calls for collections in one place -- t
 At the moment, we normalise on the client. This introduces a degree of complexity to the client-side code that, although well encapsulated, seems an unnecessary concern for the client domain -- better to have the server pass data in an ideal format and handle the details of modelling for the persistence domain.
 
 In normalising on the server, we have an additional advantage -- if the persistence model changes, for example if in the future we move to an RDS to store collection data, we can swap out the models without disturbing the client, avoiding concerns with overlapping versions etc.
+
+
+### Use 3rd party library for Drag and Drop
```

Comment:
We don't have time to make this change. But it's worth highlighting this area of the code as something to be looked at in future.